docs/sources/shared/datasources/tempo-editor-traceql.md

```diff
@@ -116,6 +130,11 @@ For more information on span details, refer to [Traces in Explore](https://grafa
 
 ![Selecting a trace ID or a span to view span details](/media/docs/grafana/data-sources/tempo/query-editor/tempo-ds-query-span-details-v11.png)
 
+Querying spansets with a large number of spans can negatively impact performance.
+You can use the **Span Limit** field in **Options** section of the TraceQL query editor.
+This field sets the maximum number of spans to return for each span set.
+If the **Span Limit** is set to `0`, then the capability is disabled.
```

Comment:
```suggestion
Querying spansets with a large number of spans can negatively impact performance.
You can use the **Span Limit** field in **Options** section of the TraceQL query editor.
This field sets the maximum number of spans to return for each span set.
By default, the maximum value that you can set for the **Span Limit** value (or the spss query) is 100. 
In Tempo configuration, this value is controlled by the `max_spans_per_span_set` parameter.
If the **Span Limit** is set to `0`, then the capability is disabled.
Entering a value higher than the default results in an error. 
```

Reply:
It looks great ✅ 

The only sentence I'm not sure is this one:
> ```If the **Span Limit** is set to `0`, then the capability is disabled.```
That's because I understood the new change as:
* If the `max_spans_per_span_set` is set to `0`, then maximum is disable (i.e. there is no maximum value that users can set. However, this can only be set by a Tempo administrator, not by the user. Whenever a maximum is set, there is no way around it.

If @joe-elliott / team agrees, I would remove this sentence.

Reply:
I just edited the statement to remove the line about capacity is disabled.